policy/modules/kernel/filesystem.te

```diff
@@ -39,6 +39,7 @@ fs_use_xattr overlay gen_context(system_u:object_r:fs_t,s0);
 fs_use_xattr squashfs gen_context(system_u:object_r:fs_t,s0);
 fs_use_xattr xfs gen_context(system_u:object_r:fs_t,s0);
 fs_use_xattr zfs gen_context(system_u:object_r:fs_t,s0);
+fs_use_xattr ubifs gen_context(system_u:object_r:fs_t,s0);
```

Comment:
Please move 2 lines up to keep alphabetical order by filesystem name.

Reply:
Thank you for pointing out this. Updated the pull request.